doc/12-icinga2-api.md

```diff
@@ -2502,6 +2503,680 @@ curl -k -s -S -i -u root:icinga -H 'Accept: application/json' \
 }
 ```
 
+## Memory Usage Analysis <a id="icinga2-api-memory"></a>
+
+The GNU libc function `malloc_info(3)` provides memory allocation and usage
+statistics of Icinga 2 itself. You can call it directly by sending a `GET`
+request to the URL endpoint `/v1/debug/malloc_info`.
```

Comment:
Is there some glibc documentation explaining how this is to be interpreted that can be referenced?

Reply:
I'd not reference anything beyond "malloc_info(3)" as the format is versioned and hence may be changed. I guess someone who wants to deeply analyse that stuff won't have any problem – looking up a manual about internals of (the current version at that time of) malloc takes just a few minutes: https://sourceware.org/glibc/wiki/MallocInternals

Reply:
Well then, why not state more clearly that the response is the `malloc_info()` result as is and add that link?